weldx/tests/transformations/test_local_cs.py

```diff
@@ -518,7 +519,8 @@ def test_interp_time_discrete_single_time():
     exp_orient = WXRotation.from_euler("x", 90, degrees=True).as_matrix()
 
     lcs_interp = lcs.interp_time("2s")
-    assert lcs_interp.time is None
+    assert lcs_interp.is_time_dependent is False
+    assert lcs_interp.time.equals(Time("2s"))
```

Comment:
this is one case where the updated behavior differs slightly

`lcs_interp.is_time_dependent` returns `False` but we can still access the (singular) timestamp from the LCS